From 945e4cd3574871016f7b30a4d41e6f87028d2b8c Mon Sep 17 00:00:00 2001
From: Gleb Mazovetskiy <glex.spb@gmail.com>
Date: Wed, 14 Oct 2020 21:11:44 +0100
Subject: [PATCH] fix-compilation

---
 attic/driver/etna_rasterizer.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/attic/driver/etna_rasterizer.h b/attic/driver/etna_rasterizer.h
index 0574391..3744fdc 100644
--- a/attic/driver/etna_rasterizer.h
+++ b/attic/driver/etna_rasterizer.h
@@ -28,5 +28,8 @@ struct pipe_context;
 
 void etna_pipe_rasterizer_init(struct pipe_context *pipe);
 
+#define VIVS_PA_SYSTEM_MODE_UNK0  0x00000001
+#define VIVS_PA_SYSTEM_MODE_UNK4  0x00000010
+
 #endif
 
-- 
2.25.1

